Yoshi.podspec

```diff
@@ -8,7 +8,7 @@
 
 Pod::Spec.new do |s|
   s.name             = "Yoshi"
-  s.version          = "1.1.0"
+  s.version          = "1.2.0"
```

Comment:
maybe this swift 3 change should be version 2.0 instead. That way if there are issues that need to be resolved in the 1.x.y line, they can be fixed.